yubikey-luks-enroll

```diff
@@ -45,7 +45,7 @@ done
 
 echo "This script will utilize slot $SLOT on drive $DISK.  If this is not what you intended, exit now!"
 
-if [ $CLEAR_SLOT -eq 1 ]; then
+if [ "$CLEAR_SLOT" -eq 1 ]; then
```

Comment:
``-eq`` is an integer comparison.
So this should be

~~~~bash
if ["$CLEAR_SLOT" = "1" ]; then
~~~~